### DISCUSSION TITLE
Why is we can't add custom schema in 'Exposed schemas' drop down?

### DISCUSSION THREAD

**Discussion by rutusfdc** [supabase/supabase#22572] [Questions] [answered]
Hi everyone,

Please help me, I am quite new in the supabase and flutterflow. How can we add custom schema in flutterflow from supabase.

Thank you in advance.

**Comment by GaryAustin1** [comment, marked as answer]
Can you clarify where you are having an issue.
![image](https://github.com/supabase/supabase/assets/54564956/17b50604-8efd-4aaf-acbf-0ce99b8d5ca2)

The dashboard allows custom schemas.

**Comment by rutusfdc** [comment]
Hi @GaryAustin1,

Thank you for helping me. Can you please check this image?I am not sure why I can't select. Do i need to change any setting or anything?
https://prnt.sc/fvLSHQpM3hcB

hope you can understand with this image.

Thank you.

Regards,

**Reply by GaryAustin1** [reply]
My guess is the UI does not know how to handle spaces.  Really bad idea to use spaces and capital letters in Postgres as they require the use of double quotes around the names.  That said I believe that is a legal name and so would be a bug you could report in supabase/supabase issues on the dashboard.
